client/src/app/site/pages/meetings/pages/motions/modules/motion-poll/components/motion-poll-form/motion-poll-form.component.ts

```diff
@@ -13,6 +14,7 @@ import {
     standalone: false
 })
 export class MotionPollFormComponent extends BasePollFormComponent {
+    public override PollPropertyVerbose: PollPropertyVerboseType = { ...this.PollPropertyVerbose, pollmethod: `Poll method` };
```

Comment:
```suggestion
    public override PollPropertyVerbose: keyof PollPropertyVerbose = { ...this.PollPropertyVerbose, pollmethod: `Poll method` };
```

Reply:
keyof PollPropertyVerbose => string
PollPropertyVerboseType => interface with pollmethod: string,....
How should your change work?

Reply:
```suggestion
    public override PollPropertyVerbose: Record<keyof PollPropertyVerbose, string> = { ...this.PollPropertyVerbose, pollmethod: `Poll method` };
```

Reply:
Also the marker method should be used on the string. 
```suggestion
    public override PollPropertyVerbose: PollPropertyVerboseType = { ...this.PollPropertyVerbose, pollmethod: _(`Poll method`) };
```

Reply:
Remove the PPVType and use Record instead.
Add the mark to the Poll method string